util/deephash/deephash.go

```diff
@@ -53,8 +53,17 @@ func (h *hasher) setBufioWriter(w *bufio.Writer) (old *bufio.Writer) {
 	return old
 }
 
+// Sum is an opaque checksum type that is comparable.
+type Sum struct {
+	sum [sha256.Size]byte
+}
+
+func (s Sum) String() string {
+	return hex.EncodeToString(s.sum[:])
+}
+
 // Hash returns the raw SHA-256 (not hex) of v.
```

Comment:
these docs could be updated now. the "not hex" part is a little old at this point.

Reply:
Done.